frontend/src/index.tsx

```diff
@@ -85,7 +85,7 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                   ),
                   children: [
                     {
-                      index: true,
+                      path: ArticleSubRoute.ARTICLES,
```

Comment:
Why we can't use `AppRoute.ARTICLES`? They look the same and lead to the same page

Reply:
I thought about it, but I think it more neat like this
@andanf-e Should I leave it in ArticleSubRoute or it's better to use AppRoute here and in redirection place?

Reply:
Let's leave it as index route